evansmusomi/burger-builder
# Handles Axios Errors Globally

*What this does*
- Adds a HOC to handle Axios errors

*How to verify*
- User can see a modal with an error message if the request to save an order fails e.g. due to network failure
